src/IBM.Cloud.SDK.Core/Authentication/Noauth/NoauthAuthenticator.cs

```diff
@@ -24,16 +24,13 @@ namespace IBM.Cloud.SDK.Core.Authentication.Noauth
     /// </summary>
     public class NoauthAuthenticator : Authenticator
     {
-        public NoauthAuthenticator(NoauthConfig config) { }
+        public NoauthAuthenticator() { }

```

Comment:
I know we have gone over the design a few times, but should we call this `NoAuthAuthenticator` instead of `NoauthAuthenticator`?

Reply:
In my view, this falls into the realm of following naming conventions for the language.  The design calls it "NoauthAuthenticator" but I think the specific capitalization should be decided based on what would be considered "idiomatic" for the particular language.